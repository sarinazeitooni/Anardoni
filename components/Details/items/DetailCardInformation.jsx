import React from 'react';
import style from './scss/DetailCardInformation.module.scss';
import messages from "./messages/messages";
import {MobileView} from 'react-device-detect';
import _ from "lodash";

function DetailCardInformation({item}) {
    const information = _.get(item, "information", ['']);
    return (
        <div className={style.preview}>
            <span className={style['preview-title']}>{messages.information}</span>
            <div className={style['about-container']}>
                {information.map((item) => {
                    return (
                        <div>
                            <div className={style.about}>
                                <div className={style['info-title']}>{item.Title}</div>
                                <span className={style['info-items']}
                                      style={{color: item.color}}>{item.description}</span>
                            </div>
                            <MobileView>
                                <div className='line'></div>
                            </MobileView>
                        </div>
                    );
                })}
            </div>
        </div>
    )
};
export default DetailCardInformation