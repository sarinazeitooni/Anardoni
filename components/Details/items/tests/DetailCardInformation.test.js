import React from 'react';
import { shallow } from 'enzyme';
import DetailCardInformation from "../DetailCardInformation";
describe("DetailCardInformation component" , ()=>{
    it("it should render a div",()=>{
        const container = shallow(<DetailCardInformation/>)
        expect(container).toMatchSnapshot();
    })
});