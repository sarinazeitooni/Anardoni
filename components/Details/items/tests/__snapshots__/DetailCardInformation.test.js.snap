// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailCardInformation component it should render a div 1`] = `ShallowWrapper {}`;
